src/View/IconView.vala

```diff
@@ -86,9 +87,10 @@ namespace FM {
             return tree as Gtk.Widget;
         }
 
-        protected override Marlin.ZoomLevel get_set_up_zoom_level () {
-            var zoom = Marlin.icon_view_settings.get_enum ("zoom-level");
-            Marlin.icon_view_settings.bind ("zoom-level", this, "zoom-level", GLib.SettingsBindFlags.SET);
+        protected override void set_up_zoom_level () {
+            Marlin.icon_view_settings.bind ("zoom-level",
```

Comment:
same as above for code style

Reply:
Theres a lot of old code where this style needs updating!